README.md

```diff
@@ -27,28 +29,39 @@ This project aims to be a strong foundation for react-native applications. It pr
 ## Usage
 
 #### Use Template button
+
 Click the "Use this template" button above the file list, then use the Owner drop-down menu, and select the account you want to own the repository. Creating a repository from a template has the following advantages:
 
 - A repository created from a template starts with a single commit.
 - Commits to a repository created from a template do appear in your contribution graph.
 - Creating a repository from a template starts a new project quickly.
 
 ### Option 1: Using React-Native-Rename
+
 You can start by cloning this repository and using [react-native-rename](https://github.com/junedomingo/react-native-rename). In the current state of this project, it should give you no issues at all, just run the script, delete your node modules and reinstall them and you should be good to go.
 
 Keep in mind that this library can cause trouble if you are renaming a project that uses `Pods` on the iOS side.
 
 After that you should proceed as with any javascript project:
+
 - Go to your project's root folder and run `npm install`.
-- Run `npm run ios` or `npm run android` to start your application!
+- If you are using xcode 12.5 or higher got to /ios and execute `pod install --repo-update`
+- Run `npm run ios:development` or `npm run android:development` to start your application!
+
+(Using yarn: `yarn ios:development` or `yarn android:development`)
```

Comment:
```suggestion
- Run `npm run ios` or `npm run android` to start your application!

(Using yarn: `yarn ios` or `yarn android`)
```

these scripts do not exist. the available options are:

- android
- android:prod
- android:staging

- ios
- ios:prod
- ios:staging